.github/workflows/js_test.yml

```diff
@@ -0,0 +1,23 @@
+name: Javascript testing
+
+on:
+  - push
+  - pull_request
+
+env:
+  TZ: Europe/London
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
```

Comment:
should we be getting the node version from the .nvmrc ?

Reply:
Oh, hadn't realised that was there. Yes, we probably should.